docs/modules/ROOT/pages/cdcExample.adoc

```diff
@@ -79,7 +79,66 @@ INFO  [main] 2022-04-11 18:47:06,433  Agent.java:92 - CDC agent started
 == Deploy Pulsar Connector
 
 . Deploy the Pulsar Cassandra Source Connector (CSC) for each CDC-enabled C* table. In the below example, we're configuring the source connector with parameters passed in `--source-config`. 
+
+The `outputFormat` source configuration controls the format of messages on the data topic. 
+
+* `key-value-avro` - Default behavior. Key and value are encoded separately in AVRO format. 
+* `key-value-json` - Key and value are encoded *separately* in JSON format.
+* `json` - Key and value are encoded *together* in a single JSON object. Key field is populated with a JSON string representing key fields. 
++
+[tabs]
+====
+Key-value-avro::
++
+--
+[source,bash]
+----
+$ pulsar-admin source create \
+  --name <csc_connector_name> \
+  --archive /pathto/to/pulsar-cassandra-source-1.0.5.nar \
+  --tenant public \
+  --namespace default \
+  --destination-topic-name <keyspace>.<table> \
+  --parallelism 1 \
+  --source-config {
+  	"events.topic": "persistent://public/default/events-<keyspace>.<table>",
+  	"keyspace": "<C*_keyspace>",
+  	"table": "<C*_table>",
+  	"contactPoints": "<C*_contact_point_list>",
+  	"port": "9042",
+  	"loadBalancing.localDc": "<C*_DC_name>"
+    "outputFormat": "key-value-avro"
+  }
+----
+--
 +
+Key-value-json::
++
+--
+[source,bash]
+----
+$ pulsar-admin source create \
+  --name <csc_connector_name> \
+  --archive /pathto/to/pulsar-cassandra-source-1.0.5.nar \
+  --tenant public \
+  --namespace default \
+  --destination-topic-name persistent://public/default/data-<keyspace>.<table> \
+  --parallelism 1 \
+  --source-config {
+  	"events.topic": "persistent://public/default/events-<keyspace>.<table>",
+  	"keyspace": "<C*_keyspace>",
+  	"table": "<C*_table>",
+  	"contactPoints": "<C*_contact_point_list>",
+  	"port": "9042",
+  	"loadBalancing.localDc": "<C*_DC_name>"
+    "outputFormat": "key-value-json"
+  }
+----
+--
++
+Json::
```

Comment:
it is worth mentioned about JSON output format the key difference (not sure if this is the right location) or maybe have a comparison table:
1/ the value for mat is JSON (not a key value schema)
2/ the key is still populated, but as a JSON string.